src/main/java/com/spring/studentmanagement/models/Role.java

```diff
@@ -23,4 +25,14 @@ public class Role {
 
     @Column(name = "role_name", length = 45, unique = true, nullable = false)
     private String roleName;
+    @OneToMany(mappedBy = "role")
+    private Collection<AppUser> appUser;
```

Comment:
We don't need bi-directional relation. We only access the role through the AppUser entity